content/chapters/compute/lab/solution/sleepy/Makefile

```diff
@@ -0,0 +1,5 @@
+BINARY = sleepy_creator
+include ../../../../../common/makefile/single.mk
+
+clean:
+	-rm -f syscalls*
```

Comment:
I think this is a leftover from previous versions?
```
$ make
Makefile:4: *** target file 'clean' has both : and :: entries.  Stop.
```
Same in `support/sleepy/Makefile`.